keyboards/handwired/ortho5x14/config.h

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xBB80
+#define PRODUCT_ID      0x050D
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    MPInc
+#define PRODUCT         Ortho 5x14
+#define DESCRIPTION     Handwired 5x14 ortholinear keyboard
```

Comment:
```suggestion
```

Reply:
Why remove this since all other handwired keyboards have this attributes?

Reply:
They don't.

Reply:
I just went ahead and removed it.